examples/dht_server.rs

```diff
@@ -227,8 +225,8 @@ fn main() {
         });
 
     let server: IoFuture<()> = Box::new(network); // TODO: remove these boxes on rustc 1.26
-    let server: IoFuture<()> = Box::new(server.select(run_server(&server_obj)).map(|_| ()).map_err(|(e, _)| e));
-    let server: IoFuture<()> = Box::new(server.select(run_lan_discovery_sender(lan_discovery_sender)).map(|_| ()).map_err(|(e, _)| e));
+    let server: IoFuture<()> = Box::new(server.select(server_obj.run()).map(|_| ()).map_err(|(e, _)| e));
```

Comment:
Remove `Box::new` if it is posssible

Reply:
As TODO says it's possible only with rustc 1.26. There was a fix for huge compilation time with long list of inner generics. And we have on travis minimal rustc 1.25.

Reply:
OK